inductiva/fluids/post_processing/openfoam.py

```diff
@@ -77,16 +104,23 @@ def get_object_pressure_field(self, save_path: types.Path = None):
             and to render it.
         """
 
-        _, object_mesh = self.get_output_mesh()
+        if self.full_output:
```

Comment:
maybe call this variable other thing like` backend_pressure_field`??

Reply:
No, this is a general parameter for all outputs and not a specific one just for pressure_field.